rhel/coalescer.go

```diff
@@ -50,6 +49,10 @@ func (*Coalescer) Coalesce(ctx context.Context, artifacts []*indexer.LayerArtifa
 	}
 	// The same thing has to be done in reverse, because the first layer(s) are missing
 	// the relevant information.
+	//
+	// With [ENGCMP-5332], this shouldn't be needed, so check back in 5 years.
+	//
+	// [ENGCMP-5332]: https://issues.redhat.com/browse/ENGCMP-5332
```

Comment:
looks like it's closed as of late April

Reply:
Right, so we should be able to assume it's everywhere in like 5 years